src/services/member/plugins/thumbnail/schemas.ts

```diff
@@ -27,7 +27,7 @@ export const download = {
     replyUrl: Type.Boolean({ default: false }),
   }),
   response: {
-    [StatusCodes.OK]: {},
+    [StatusCodes.OK]: Type.String({ description: 'Url string of the avatar if replyUrl is true' }),
```

Comment:
Not to be changed now, but do we really need to support replying with the file as a blob ? i.e. can we remove the `replyUrl` param and always rely with the url instead ?

Reply:
Yes, I think we never use one of the case.